includes/API/AdPartner/BusinessApi.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * API module for managing Ad Partner businesses.
+ *
+ * This class provides an interface for interacting with the Ad Partner's
+ * Business API via WooCommerce Connect Server (WCS). It supports operations
+ * such as listing available businesses for the authenticated merchant and
+ * retrieving details for a specific business.
+ *
+ * Requests are proxied through WCS, which handles secure authentication
+ * and communication with the Ad Partner's remote API.
+ *
+ * @since 0.1.0
+ * @package RedditForWooCommerce\API\AdPartner
+ */
+
+namespace RedditForWooCommerce\API\AdPartner;
+
+use RedditForWooCommerce\API\AdPartner\BaseAdPartnerApi;
+use RedditForWooCommerce\Utils\Storage\Options;
+use RedditForWooCommerce\Utils\Storage\OptionDefaults;
+use WP_Error;
+
+/**
+ * API module for managing Ad Partner businesses.
+ *
+ * This class provides methods to fetch a list of businesses available
+ * to the authenticated merchant and to retrieve detailed information
+ * for the currently selected business.
+ *
+ * @since 0.1.0
+ */
+class BusinessApi extends BaseAdPartnerApi {
+	/**
+	 * Retrieves the list of businesses for the current merchant.
+	 *
+	 * This method calls the Ad Partner API to return all businesses
+	 * associated with the authenticated account. The request is proxied
+	 * through WCS for authentication.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @return \WP_REST_Response|WP_Error REST response containing the list of businesses
+	 *                                    or error if the request fails.
+	 */
+	public function list() {
+		return $this->wcs->proxy_get(
+			'/v3/me/businesses'
```

Comment:
```suggestion
			'ads/me/businesses'
```

Please replace `/v3/ENDPOINT` with `/ads/ENDPOINT`

Please do this for all `v3` endpoints.

Conversions should be `conversions/events/{{pixel_id}}`